Makefile

```
@@ -81,6 +82,9 @@ license: LICENSE.md
 
 regen-apis: | clean-all-api clean-all-cli gen-all-api gen-all-cli license
 
+gen-test: $(PYTHON)
```

Firs of all, I love all of this!
When reading the make-target for the first time, I thought it would be generating tests, even though in fact it is running them.
Renaming the target would do the job - this is just a suggestion:
```suggestion
test: $(PYTHON)
```